tests/integration/sugar/test_wallet.py

```diff
@@ -83,6 +84,110 @@ def test_generate_faucet_wallet_custom_host_sync_json_rpc(self):
         )
         self.assertTrue(int(result.result["account_data"]["Balance"]) > 0)
 
+    async def test_generate_faucet_wallet_testnet_async_websockets(self):
```

Comment:
Why are all of these new tests needed? Seems unnecessary and will just make the tests take longer

Reply:
xrpl-js has integration tests for every single testnet/devnet. Figured xrpl-py should have them too. Should I remove them?

Reply:
No, that's fine